hadoop-yarn-project/hadoop-yarn/hadoop-yarn-ui/src/main/webapp/app/serializers/yarn-jhs-container.js

```diff
@@ -45,9 +45,13 @@ export default DS.JSONAPISerializer.extend({
   },
 
   normalizeArrayResponse(store, primaryModelClass, payload/*, id, requestType*/) {
-
     payload = payload["containerLogsInfo"]
 
+    if (payload && !Ember.isArray(payload) && !Ember.isEmpty(payload)) {
```

Comment:
Nit: a cleaner (and more JS like :D) solution would be to simply say 
`payload = [].concat(payload["containerLogsInfo])`
This way there is no need for the custom isEmpty/isArray methods.